weekly-updates/weekly-update.2015-09-11.md

```diff
@@ -0,0 +1,71 @@
+* **node-gyp**: Float v3.0.3 which has improved support for Node.js and io.js v0.10 to v4+ (Rod Vagg) [#2700](https://github.com/nodejs/node/pull/2700)
+* **npm**: Upgrade to version 2.14.3 from 2.13.3, includes a security update, see https://github.com/npm/npm/releases/tag/v2.14.2 for more details, (Kat Marchán) [#2696](https://github.com/nodejs/node/pull/2696).
+* **timers**: Improved timer performance from porting the 0.12 implementation, plus minor fixes (Jeremiah Senkpiel) [#2540](https://github.com/nodejs/node/pull/2540), (Julien Gilli) [nodejs/node-v0.x-archive#8751](https://github.com/nodejs/node-v0.x-archive/pull/8751) [nodejs/node-v0.x-archive#8905](https://github.com/nodejs/node-v0.x-archive/pull/8905)
+
+### Known issues
+
+See https://github.com/nodejs/node/labels/confirmed-bug for complete and current list of known issues.
+
+* Some uses of computed object shorthand properties are not handled correctly by the current version of V8. e.g. `[{ [prop]: val }]` evaluates to `[{}]`. [#2507](https://github.com/nodejs/node/issues/2507)
+* Some problems with unreferenced timers running during `beforeExit` are still to be resolved. See [#1264](https://github.com/nodejs/node/issues/1264).
+* Surrogate pair in REPL can freeze terminal. [#690](https://github.com/nodejs/node/issues/690)
+* Calling `dns.setServers()` while a DNS query is in progress can cause the process to crash on a failed assertion. [#894](https://github.com/nodejs/node/issues/894)
+* `url.resolve` may transfer the auth portion of the url when resolving between two full hosts, see [#1435](https://github.com/nodejs/node/issues/1435). 
+
+### Node.js Foundation Combines Node.js and io.js Into Single Codebase in New Release
+
+Node.js Foundation posts an entry about Node.js new release and introduction of Node Foundation. [The detail is here](https://nodejs.org/en/blog/announcements/foundation-v4-announce/)
+
+### Node.js Interactive
+
+[Node.js Interactive](http://events.linuxfoundation.org/events/node-interactive) is a new, annual, vendor-neutral conference for Node.js.  It is being led by the newly formed Node.js Foundation.
+
+### Community Updates
+
+* We *still* need a V8 maintainer for our LTS build! Head on over [to GitHub](https://github.com/nodejs/LTS/issues/28) to see if the requirements match your capabilities.
+* According to [Raygun blog](https://raygun.io/blog/2015/09/nodejs-and-io-js-are-now-one/), Node.js v4.0.0 is 8% faster than Node.js v0.12.
+* Tony Pujals, evangelism WG member, posts an [entry about Node.js v4.0.0](https://www.linkedin.com/pulse/node-v400-here-tony-pujals), He introduced about ES6 on Node.js v4.0.0, LTS, and SemVer.
+
+If you have spotted or written something about Node.js and io.js, do come over to our [Evangelism team repo](https://github.com/nodejs/evangelism) and suggest it on the [Issues page](https://github.com/nodejs/evangelism/issues), specifically the Weekly Updates issue.
```

Comment:
Nit: `Node.js or io.js` rather than `Node.js and io.js`. Perhaps even better, just `Node.js` and don't mention io.js specifically at this point.